boot/zephyr/Kconfig

```diff
@@ -303,6 +303,21 @@ endif
 
 endchoice
 
+config BOOT_SIGNATURE_USING_KMU
+	bool "Use KMU stored keys for signature verification"
+	default n
```

Comment:
```suggestion
```
Never needed

Reply:
Eh, and this is another thing I can not learn not do do.